src/main/java/ru/itmo/sd/deadliner2bot/utils/chrono/DateTimeUtils.java

```diff
@@ -31,10 +30,25 @@
             .parseDefaulting(ChronoField.HOUR_OF_DAY, 0)
             .parseDefaulting(ChronoField.MINUTE_OF_HOUR, 0)
             .toFormatter();
-    public static final LocalDateTime startDateUnconfirmed = LocalDateTime.of(2028, 12, 31, 23, 59);
-    public static final LocalDateTime startDateConfirmed = LocalDateTime.of(1995, 12, 31, 23, 59);
 
-    public static LocalDateTime parseDateTime(String dateTimeString) {
+    private LocalDateTime stagingWeekStartDateTime;
+    private LocalDateTime stagingWeekAuxStartDateTime;
+
+    @Value("${notifications.daily.staging-week}")
+    private void setStagingWeekStartDateTime(String stagingWeek) {
+        if (stagingWeek != null && !stagingWeek.isEmpty()) {
+            stagingWeekStartDateTime = LocalDate.parse(stagingWeek).atStartOfDay().minusDays(1);
+        }
+    }
+
+    @Value("${notifications.daily.staging-week-aux}")
+    private void setStagingWeekAuxStartDateTime(String stagingWeekAux) {
+        if (stagingWeekAux != null && !stagingWeekAux.isEmpty()) {
+            stagingWeekAuxStartDateTime = LocalDate.parse(stagingWeekAux).atStartOfDay().minusDays(1);
+        }
+    }
```

Comment:
I cannot understand purpose of changing this costants to external variables.

Reply:
It's better to keep all non-trivial parameters in a single place for easier configuration and shared access.

Reply:
I agree. Question answered.